### ISSUE TITLE
Slider bug when changing value

### ISSUE THREAD

**Issue by 3lwan** [MauriceConrad/Photon#13]
I am binding the slider to a variable using angularjs. When I change the value the slider's position change correctly, howere the colour is not updated as shown in the image. 
![2017-12-13 13_45_19-display-objects-side-panel html](https://user-images.githubusercontent.com/20580352/33939623-2f20819a-e00c-11e7-87dc-d8cfcee31f0f.png)

Am i missing something or is it a bug?
Thanks

**Comment by ElyaConrad** [comment]
@3lwan Thanks for this point!

I didn't saw this use case and therefore there was no solution but now I've implemented a solution ;-)
It didn't worked before because the `sliderVisualCalc()` method was only called when an "input" event fires but not when the `value` of the slider changes (Which is the way Angular works). Now I added a special handle for the `[Setter]` of each Input type range element that fires the `sliderVisualCalc` every time the value changes.

Look at the new code :)
